docs/SOAP-Service-Methods.md

```diff
@@ -167,15 +167,16 @@ Get list of all applications that are present in this PowerAuth Server instance.
 
 ### Method 'getApplicationDetail'
 
-Get detail of application with given ID, including the list of versions.
+Get detail of application with given ID or name, including the list of versions.
```

Comment:
I am considering whether having a separate SOAP method would not be better than updating the `getApplicationDetail` method. I will think about it. @petrdvorak, what do you think?

Reply:
Well, for the current `getApplicationDetail` and its usage I see the id or name approach as the right one. But for sure as simple and fully clear approach two `getApplicationDetail` and `getApplicationDetailByName` will meet the requirements too. This case is a little bit not clear.